crates/client/tests/app.rs

```diff
@@ -22,6 +23,24 @@ use std::{
 
 mod fixture;
 
+fn make_agent(space: kitsune2_api::SpaceId) -> String {
```

Comment:
I suggest to DRY this function to a shared place for both app and admin websocket.

Reply:
So initially i did that by putting it in Fixture, but then clippy complained in the CI checks that it was unused, so I just duplicated it.  How do I fix that?

Reply:
My suggestion above might mean that a common function isn't needed for this. If it is, the solution is common modules for integration tests -> https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html